coforma/swift-tech-challenge
# Issue/initial repo setup

# Description 
Initial Repo Scaffolding
Closes #1 

# To Verify:
 /documentation/design exists
 /documentation/research exists
 /documentation/engineering exists
 /documentation/engineering/adrs exists
 .github exists with issue templates
 License file exists
 Verify labels
Commit signatures are in place
